### ISSUE TITLE
Participation in the 3D world

### ISSUE THREAD

**Issue by theo-armour** [Abantech/abantech.github.io#3]
@GMelencio @jamesAbantech 

Does anybody here a member of any 3D type forums or regularly visit or read anything regularly regarding 3D, Leap, VR, NUI etc?

Places I go to include

https://community.leapmotion.com/
https://www.reddit.com/r/webgl/
https://www.reddit.com/r/threejs/
https://plus.google.com/u/0/+ThreejsOrg
https://twitter.com/mrdoob

I don't visit these places as quite frequently as I would like to. Amd I'm sure you don't surf as much as you'd like to either. ;-(

But if we all do some looking and sharing of findings - we can help Abntech stay at the sharp edge of things.

Please share some of the places you go to.

Where to keep this data - and how to keep it up to date - is another issue. And we'll get to that

**Comment by theo-armour** [comment]
Anybody here?

**Comment by GMelencio** [comment]
Yes, I agree we should post them here. Like I said James and I talk and
share such stuff regularly on whatsapp or in person but it would be better
to share this stuff to the rest of the team. We may not have been so
insides by OSVR if so.

Soeaking of, did you see James' new contribution to the start of our c++
library? Complete with readme.md? Self-documenting (a-la javadoc) c++ code.
Imagine how much easier it will be to share knowledge about our code to
ourselves and to the world. Hooray for James & Docxygen ftw!

**Comment by theo-armour** [comment]
@GMelencio 

And the link to whatever you are talking about is where???

The main thing is that you are participating in an online dialog. Thank you are this.

**Comment by GMelencio** [comment]
good call, I missed that - here is the link to what I was talking about:
https://github.com/Abantech/EfficioCPP

The other links are embedded therein.

btw, @jamesAbantech built it so that even if you dont have the external
reference it will still compile (nice work!)

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688

**Comment by theo-armour** [comment]
@GMelencio 

All cool.

And let's continue CPP discussions in CPP issues - and return to the original topic of this thread which is about: where and how / can and should the team gather and share new information regarding ideas and strategies for HCI/NUI as it relates to Abantech.

Team members should agree to cover certain sites and report back to the group when a topic of interest appears. We should all be following different sites (so the overlap is small ).

We should share what sites we look at regularly and if other team members suggest we add another site or two to look at that's OK.

So what are you looking at?

PS my list is in the first message of this issue.

**Comment by GMelencio** [comment]
@Abantech/core

I must confess I am not as up to date as I would like to be. This should be
something I/we do as a routine ("sharpen the saw"). I mostly end up looking
at the things that are pointed out by the people I meet that are into this
- so there's no one place in general. However, I have compiled a short list
  of sites that I intend to visit more often - these are below:

http://www.notey.com/blogs/gesture-control
http://www.augmented.org/blog/
http://gigantico.squarespace.com/
https://gamesalfresco.com/
https://www.reddit.com/r/VirtualReality
http://www.roadtovr.com/
http://uploadvr.com/
http://vrscout.com/
http://www.vrfocus.com/

Mostly on VR but more to come on the 3D side of things

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688

**Comment by theo-armour** [comment]
@GMelencio 

Your list of VR links is cool. But would any of us change our current reading habits and actually add any of these sites to our daily reads?? I kind of doubt it.

Which comes back to the original topic:

> Please share some of the places you go to.

And by this I was wondering what you read now - as opposed to what you think you should be reading [but most likely never will].

@GMelencio my guess ids that you take in a lot og Beltway VC and angel investing readings.

The point of this would be to see if we could channel some of the reading we do 'naturally' into something that could benefit Abantech?

For example, we might find a good Beltway hackathon to participate in or whatever.

In any case 99% of what we read will be of no use to Abantech. But what do we do with the 1% that might be useful?

**Comment by theo-armour** [comment]
of possible interest

https://github.com/nickgillian/grt

http://www.nickgillian.com/

**Comment by GMelencio** [comment]
I've been meaning to answer this. I guess the answer is "not any one
place". I do stay busy and try to keep up to date as much as possible, but
I simply don't go to any one place to read stuff or find information. Wiht
the advent of targetted articles from Twitter and Facebook as well as the
topics I see from the groups at meetup.com for which I am a member - I have
plenty to read.

A good example is that on Twitter, I just found out the Mr. Doob recently
pickedup the HTC Vive and made a tiltbrush-like app (like I did) just a day
before.

On the meetup groups (HacDC) I learned about this, which might be of
interest to you: ImplicitCAD. FOSS 3D modeling specialized for 3D Printing
http://hackaday.com/2016/04/18/implicitcad-programmatic-cad-built-with-3d-printing-in-mind/
- it ensures that all the rendered objects are manifold and bypasses
  mesh-generation and lets things be "slices"

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688

**Comment by GMelencio** [comment]
Oh and, no: I don't read hardly anything about Beltway VC's or Angel
investors. I probably should but I have yet to drum up the interest to do
so or the time to make for it.

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688

**Comment by theo-armour** [comment]
@GMelencio 

The topic here is about how we as a group gain new information - in particular information relating to 3D/VR etc.

The mission would be to cover more bases by reducing overlaps and adding more appropriate sources to our information gathering regimens.

I provided a list of sites I tend to check out during the day or week,

Does anybody else have any places they regularly visit for news or insights? Or is all data gathered in some _ad hoc_ manner? (Though even there is still some methods and results that can be shared for mutual benefit).

We are suffocating from a lack of product vision, clarity and specification. A coordinated input of ideas could help...

**Comment by theo-armour** [comment]
@GMelencio 

A light bulb just lit up!

Greg: do you receive a lot of your new data via YouTube and other video streams?

**Comment by GMelencio** [comment]
@Theo-Armour

I recieve them via twitter/facebook and some via youtube. But yes, I do get
a good amount of what I truly get into via video.

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688

**Comment by theo-armour** [comment]
So are there any particular YouTube channels that you follow? Or other sources?

And should we be thinking about adding to the mix and of having some place where you can record links to significant finds?

**Comment by GMelencio** [comment]
I typically follow links down from something I've recently seen. The
problem is that not all posters go through the effort of creating
"channels" so the subscription sometimes tends to be a hit an miss (getting
better but not there yet.

As for interesting links on your tube et all - yes we  do - it's on Slack!

Greg Melencio
CEO and Founder
Abantech LLC
571-402-4688
